python/cudf/cudf/core/dtypes.py

```diff
@@ -125,9 +125,9 @@ class CategoricalDtype(_BaseDtype):
     stored on the GPU.
     """
 
-    ordered: Optional[bool]
+    ordered: bool
 
-    def __init__(self, categories=None, ordered: bool = None) -> None:
+    def __init__(self, categories=None, ordered: bool = False) -> None:
```

Comment:
If this API is public, we should improve its docstring to look more like https://pandas.pydata.org/docs/reference/api/pandas.CategoricalDtype.html.

Reply:
Updated the docstring, but a quick lookup shows this isn't published in docs yet. Added an issue to follow-up: https://github.com/rapidsai/cudf/issues/11605

Reply:
Thanks!